renderer/test/js_ethereum_provider_browsertest.cc

```diff
@@ -89,6 +94,13 @@ class JSEthereumProviderBrowserTest : public InProcessBrowserTest {
     ASSERT_TRUE(https_server_.Start());
     ASSERT_TRUE(test_server_handle_ =
                     embedded_test_server()->StartAndReturnHandle());
+    auto* keyring_service =
+        brave_wallet::KeyringServiceFactory::GetServiceForContext(
+            browser()->profile());
+    // Create wallet since native wallet should not be injected otherwise
+    keyring_service->CreateWallet("password", base::DoNothing());
```

Comment:
Instead of creating wallet that we don't need in test , we should just change the prefs to BraveWallet.

Reply:
Done